### PR TITLE
fix: DEV-3693: Explore reviews module show buggs

### PR DESCRIPTION
**Pull request by yyassi-heartex**

### CODE REVIEW

File: e2e/tests/unfinished-polygons.test.js [hunk 1]
```diff
@@ -103,9 +103,12 @@ Scenario("Saving polygon drawing steps to history", async function({ I, LabelStu
...
   I.say(`try to undo closing and 2 last points`);
-  I.pressKey(['CommandOrControl', 'Z']);
-  I.pressKey(['CommandOrControl', 'Z']);
-  I.pressKey(['CommandOrControl', 'Z']);
+  I.click("button[aria-label=Undo]");
+  I.click("button[aria-label=Undo]");
+  I.click("button[aria-label=Undo]");
+  // I.pressKey(['CommandOrControl', 'Z']);
+  // I.pressKey(['CommandOrControl', 'Z']);
+  // I.pressKey(['CommandOrControl', 'Z']);
```
Comment (bmartel) [inline e2e/tests/unfinished-polygons.test.js:111]: Are these meant to stay or be replaced strictly by the button usage?

Maybe can we instead split to another test to ensure both work correctly in the same fashion?

**Review by bmartel — CHANGES_REQUESTED** [review]
Looks good, just one suggestion with the unfinished polygon's test.

File: e2e/tests/unfinished-polygons.test.js (hunk 1 above)
Comment (yyassi-heartex) [reply e2e/tests/unfinished-polygons.test.js:111]: they should be removed, as currently the hotkey undo is not working the same way as clicking the undo button and it should be addressed in another ticket

**Review by bmartel — APPROVED** [review]

**Review by bmartel — APPROVED** [review]

**Review by bmartel — APPROVED** [review]

**Pull request merged**